resources.md

```diff
@@ -10,7 +10,7 @@ It's often said that the best way to learn to write programs is to write program
 
 ## Python-specific exercises
 
-- [exercism](https://exercism.io/): <q>Code practice and mentorship for everyone</q>, with over 100 [Python-specific exercises](https://exercism.io/tracks/python).
+- [exercism](https://exercism.io/): Crowd-sourced <q>code practice and mentorship for everyone</q>, with over 100 [Python-specific exercises](https://exercism.io/tracks/python). Installing their command-line interface can be a little complicated, but after that the submission system is smoother than many online exercise portals. Beginners should be aware that some problem descriptions may be incomplete or confusing. Feel free to ask for help on the slack if a problem seems unclear!
```

Comment:
```suggestion
- [exercism](https://exercism.io/): Crowd-sourced <q>code practice and mentorship for everyone</q>, with over 100 [Python-specific exercises](https://exercism.io/tracks/python). Installing their command-line interface can be a little complicated, but after that the submission system is smoother than many online exercise portals. Some exercise descriptions may be incomplete or confusing.
```

Thanks for taking my suggestion; here's another one. ;)

I like the intention of "ask for help", but it could apply to any of these resources, and not all the readers will know what "the slack" is.

How about adding a sentence like this after the first paragraph of these page?

```
If you need help with a specific exercise, feel free to ask on our [Slack workspace](https://about.bostonpython.com/slack).
```